src/routes/Generator.svelte

```diff
@@ -38,24 +38,43 @@
 			};
 	}
 
-	async function exportArt() {
+	async function renderArt() {
 		const { Image } = await import('image-js');
 		const canvas = await html2canvas(albumArt);
 
-		let image = await Image.load(canvas.toDataURL('image/jpeg'));
+		let image = await Image.load(canvas.toDataURL('image/png'));
 		const originalWidth = image.width;
 		image = image
 			.resize({ preserveAspectRatio: true, width: originalWidth * 0.85 }) // Get those compression artifacts like the original cover
 			.resize({ preserveAspectRatio: true, width: originalWidth })
-			.blurFilter({ radius: 2 });
+			.blurFilter({ radius: 3 });
+
+		return image;
+	}
+
+	async function downloadArt() {
+		const image = await renderArt();
 
 		const link = document.createElement('a');
 		link.href = image.toDataURL();
 
-		link.download = `${text}.jpeg`;
+		link.download = `${text}.png`;
 		link.click();
 	}
 
+	async function copyArtToClipboard() {
+		const image = await renderArt();
+		const blob = await image.toBlob('image/png', 1.0);
+
+		const item = new ClipboardItem({
+			'image/png': blob
+		});
+
+		await navigator.clipboard.write([item]);
+
+		alert('Copied image to clipboard');
```

Comment:
I think if we could show a message below the buttons that'd be great if you want an alternative to this.